charts/opensrp-web/values.yaml

```diff
@@ -89,7 +89,7 @@ containerEnvironmentVariables:
   EXPRESS_OPENSRP_CLIENT_SECRET: "{{ .Values.sharedVars.opensrpClientSecret }}"
   EXPRESS_OPENSRP_LOGOUT_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/logout.do"
```

Comment:
do we need to also change this?

Reply:
That is ok, as it is, some deployments will still use this value as it is